.github/draft_release_notes.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/sh
+
+# this script assumes the GITHUB_TOKEN and PREVIOUS_TAG environment variables have been set;
+# it produces a 'Changes.md' file as its final output;
+# the file 'last-300-prs-with-release-note.txt' that is produces is intermediate data; it is not
+# pruned for now to assist development of the release notes process (we are still curating all this)
+
+sudo apt-get -y install jq
+sudo apt-get -y install wget
+curl -L https://github.com/github/hub/releases/download/v2.14.2/hub-linux-amd64-2.14.2.tgz | tar xzf -
+PWD=`pwd`
```

Comment:
Another `shellcheck` suggestion: Do not use legacy backticks style anymore.
```suggestion
PWD="$(pwd)"
```

Reply:
sure I'll pull that into the next push